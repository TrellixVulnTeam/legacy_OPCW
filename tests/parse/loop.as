loop {
    do_stuff();
    do_more_stuff();
    break;
    continue;
}
