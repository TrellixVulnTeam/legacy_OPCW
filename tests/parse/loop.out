ModuleDecl <?> _
  Nodes <?> 
    Loop <?> 
      Block <?>
        Nodes <?> 
          CallExpr <?>
            Ident <?> do_stuff
            Arguments <?> 
          CallExpr <?>
            Ident <?> do_more_stuff
            Arguments <?> 
          Break <?> 
          Continue <?> 
